Product/Production/Services/DirectCore/src/main/java/gov/hhs/fha/nhinc/direct/DirectAdapter.java

```diff
@@ -109,12 +109,12 @@ private URL getUrl() {
             urlString = oProxyHelper
                     .getAdapterEndPointFromConnectionManager(DIRECT_CONFIG_SERVICE_NAME);
         } catch (ConnectionManagerException ex) {
-            LOG.error("Error reading directConfig Url cannot be found: " + ex.getMessage());
+            LOG.error("Error reading directConfig Url cannot be found: {}", ex.getLocalizedMessage(), ex);
```

Comment:
I am OK with just using ex.getErrorMessage() than getLocalizedMessage(), but I do agree that we should try to stick with one method in our code for loggers.